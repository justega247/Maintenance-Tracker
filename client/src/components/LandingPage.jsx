import React from 'react';
import { Link } from 'react-router-dom';
import Header from './Header';
import Button from './common/Button';
import { frontendRoutes } from '../constants/routes';

const LandingPage = () => (
  <div>
    <Header />
    <div className="landing__image">
      <div className="landing__quote">
        <h1>No need to wait till something breaks</h1>
      </div>
      <div className="landing__button">
        <Link to={frontendRoutes.SIGN_IN}>
          <Button className="button button--login" text="login" />
        </Link>
        <Link to={frontendRoutes.SIGN_UP}>
          <Button
            className="button button--signup"
            text="create your account"
          />
        </Link>
      </div>
    </div>
  </div>
);

export default LandingPage;
