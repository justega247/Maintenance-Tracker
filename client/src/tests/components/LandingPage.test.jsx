import React from 'react';
import { shallow } from 'enzyme';
import LandingPage from '../../components/LandingPage';

describe('Landing Page', () => {
  test('should correctly render LandingPage', () => {
    const wrapper = shallow(<LandingPage />);
    expect(wrapper).toMatchSnapshot();
  });
});
