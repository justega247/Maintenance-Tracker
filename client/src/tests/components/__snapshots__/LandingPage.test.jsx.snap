// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing Page should correctly render LandingPage 1`] = `
<div>
  <Header />
  <div
    className="landing__image"
  >
    <div
      className="landing__quote"
    >
      <h1>
        No need to wait till something breaks
      </h1>
    </div>
    <div
      className="landing__button"
    >
      <Link
        replace={false}
        to="/login"
      >
        <Button
          className="button button--login"
          text="login"
          type="button"
        />
      </Link>
      <Link
        replace={false}
        to="/signup"
      >
        <Button
          className="button button--signup"
          text="create your account"
          type="button"
        />
      </Link>
    </div>
  </div>
</div>
`;
